src/scripts/breeding/Breeding.ts

```diff
@@ -223,7 +223,11 @@ class Breeding implements Feature {
     }
 
     public gainEggSlot() {
-        this.eggSlots += 1;
+        if (this.eggSlots < 4){
+            this.eggSlots += 1;
+        }
+    } else {
+        this.eggSlots += 0;
```

Comment:
Redundant

Reply:
redundant?

Reply:
Why increase a variable y 0, it is not needed

Reply:
fixed